README.md

```diff
@@ -1,3 +1,8 @@
+# Status
+
+[![Coverage Status](https://coveralls.io/repos/github/andela/ah-shakas/badge.svg?branch=develop)](https://coveralls.io/github/andela/ah-shakas?branch=develop)
```

Comment:
can you change this to master? I'm gonna explain a bit later why

Reply:
Yeah, sure.

Reply:
@FawazFarid I have made the proposed changes.